whamester/WMDD-4885-Project
# Whamester/fix 96 view hazards hazard detail api reactions and flag report return error

fix #96 

How to test the solution?
- Go to the homepage and click on a report that someone else has created. You should be able to see the three actions available.
- Reactions Still there and not there, clicking on either should save the reaction, display the notification alert and remove the button from the card
- Flag a report, click on the `Flag as fake` button should trigger save the action, display the notification alert and remove the button from the card
